metadata-ingestion/src/datahub/utilities/sqlglot_lineage.py

```diff
@@ -238,6 +253,7 @@ def __init__(
         self.platform = platform
         self.platform_instance = platform_instance
         self.env = env
+        self.urns: Optional[Set[str]] = None
```

Comment:
Optimization 1: don't fetch schema metadata for urns that don't exist, which I expect could be a lot e.g. when we filter out urns. This doesn't apply if we're filtering out temporary tables like with bigquery